com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -915,12 +915,24 @@ static float GetLensFlareLightAttenuation(Light light, Camera cam, Vector3 wo)
 
         void DoLensFlareDatadriven(Camera camera, CommandBuffer cmd, RenderTargetIdentifier source, bool usePanini, float paniniDistance, float paniniCropToFit)
         {
+            var nonJitteredCameraProj = camera.projectionMatrix;
+            var cameraProj = nonJitteredCameraProj;
+            var gpuProj = GL.GetGPUProjectionMatrix(cameraProj, false);
```

Comment:
These vars are not used anywhere?

Reply:
some used to produce:
gpuVP used bellow.
I'll remove the unused one.